packages/jest-worker/src/child.js

```diff
@@ -61,7 +65,7 @@ function reportSuccess(result: any) {
   process.send([PARENT_MESSAGE_OK, result]);
 }
 
-function reportError(error: Error) {
+function reportError(error: Error, type?: number = PARENT_MESSAGE_ERROR) {
```

Comment:
In line 129, you do `reportError(error, PARENT_MESSAGE_INITIALIZE_ERROR)`; however `PARENT_MESSAGE_INITIALIZE_ERROR` is incompatible with `PARENT_MESSAGE_ERROR`.

Also, `type` must not be optional.